playwright.config.ts

```diff
@@ -29,7 +29,11 @@ export default defineConfig({
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
     trace: "on-first-retry",
   },
-  timeout: process.env.CI ? 60000 : 30000,
+  timeout: process.env.CI ? 60_000 : 30_000,
+
+  expect: {
+    timeout: process.env.CI ? 10_000 : 8_000,
```

Comment:
As noted in https://github.com/open-sauced/app/issues/4072

> we should consider extending the playwright "assertions" timeout:

> await expect(locator).toBeHidden({ timeout: 10000 });

> This is different from the global timeout we have configured:

> timeout: process.env.CI ? 60000 : 30000,

From what I understand of how to configure playwrite, modifying the global timeout won't modify the timeout of the individual assertions that are timing out after 5 seconds (the default assertion timeout).

> Web-first assertions like expect(locator).toHaveText() have a separate timeout, 5 seconds by default. Assertion timeout is unrelated to the test timeout.

https://playwright.dev/docs/test-timeouts#expect-timeout

Reply:
Right, I added the `expect.timeout` to the Playwright config which modifies each assertion timeout. The global `timeout` is still the same as it was